coalib/settings/ConfigurationGathering.py

```diff
@@ -143,6 +143,7 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
 
     :param arg_list:    The list of command line arguments.
     :param log_printer: The LogPrinter object for logging.
+    :param arg_parser:  An ``ArgParser`` object.
```

Comment:
Yes, John, please fix just this one line as @userzimmermann has requested.
Someone else will fix all the others.